first_order_model/shrink_util.py

```diff
@@ -1181,7 +1247,7 @@ def try_reduce(curr_loss, curr_model, dataloader, layer_graph,
 
 def reduce_macs(model, target, current, kp_detector, discriminator,
                 train_params, dataloader, metrics_dataloader, generator_type,
-                lr_size, generator_full, sort, steps_per_it, device_ids, log_dir, discriminator_full):
+                lr_size, generator_full, sort, steps_per_it, device_ids, log_dir, discriminator_full, image_shape):
```

Comment:
where are you using the image shape?

Reply:
also line too long

Reply:
It is used in MACS calculation. Line 1161.